examples/build.sh

```
@@ -0,0 +1,50 @@
+  echo "Invalid command provided: '$1'. Please provide 'build' or 'run' as the command."
+  exit 1
+  ;;
+esac
+
+# Pack the local ballerina/edi module and push it to the local repository so the
+# examples build against the in-repo changes rather than the published version.
+echo "Packing and pushing the local ballerina/edi module to the local repository"
+(cd "$BAL_MODULE_DIR" && bal pack && bal push --repository=local)
+
+# Build (or run) each example against the local module.
+for dir in $(find "$BAL_EXAMPLES_DIR" -mindepth 1 -maxdepth 1 -type d); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)